google-apis-common/src/serde.rs

```diff
@@ -160,8 +162,8 @@ pub mod urlsafe_base64 {
         where
             D: Deserializer<'de>,
         {
-            let s: &str = Deserialize::deserialize(deserializer)?;
-            base64::decode_config(s, base64::URL_SAFE).map_err(serde::de::Error::custom)
+            let s: Cow<'de, String> = Deserialize::deserialize(deserializer)?;
```

Comment:
Can you elaborate why this is necessary?

Reply:
So when you try to deserialize from a file reader, serde is not happy and says it expected an owned string.